src/backend/routers/paras.ts

```diff
@@ -6,7 +6,6 @@ export const paraProcedures = {
   getParaById: procedure
     .input(z.object({ user_id: z.string().uuid() }))
     .query(async (req) => {
-      console.log("req", req);
```

Comment:
Did you want to leave this in there?

Reply:
This is entirely due to our team leaving it there, sorry. Do you want to comment it out or delete it? It's no longer there in the newer PR for archivePara